src/main/java/com/xpanxion/java/assignments/student1/Worker1.java

```diff
@@ -69,6 +69,21 @@ public void ex6() {
         Collections.sort(cats, new com.xpanxion.java.assignments.student1.Sortbyname());
         System.out.println(cats);
     }
+
+    public void ex7() {
+        HashMap<String, Integer> wordMap = new HashMap<>();
+        var words = DataAccess.getWords().split(" ");
+
+        Arrays.stream(words)
```

Comment:
Awesome job, but does this print the list in alphabetical order?

Reply:
Ooooohhh didn't see that part